backend/go.mod

```diff
@@ -3,20 +3,17 @@ module github.com/cilium/hubble-ui/backend
 go 1.14
 
 require (
-	github.com/cilium/cilium v1.8.2
-	github.com/fatih/color v1.7.0
-	github.com/golang/protobuf v1.4.2
-	github.com/kardianos/osext v0.0.0-20170510131534-ae77be60afb1 // indirect
-	github.com/shirou/w32 v0.0.0-20160930032740-bb4de0191aa4 // indirect
+	github.com/cilium/cilium v1.8.3
+	github.com/golang/protobuf v1.4.1
```

Comment:
any reason to downgrade to 1.4.1 from 1.4.2 ?

Reply:
I just removed all requires and leave the cilium one and performed the `go mod tidy`

Reply:
oh right, cilium branch v1.8 is not using protobuf v1.4.2 as per master.